content/manuals/billing/cycle.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Change your billing cycle
+weight: 50
+description: Learn to change your billing cycle for your Docker subscription
+keywords: billing, cycle, payments, subscription
+aliases:
+- /billing/cycle/
+---
+
+You can pay for a subscription plan on a monthly or yearly billing cycle. You select your preferred billing cycle when you buy your subscription.
+
+> [!NOTE]
+>
+> Business plan is available only on yearly billing cycle.
+
+If you have a monthly billing cycle, you can choose to switch to an annual billing cycle.
+
+> [!NOTE]
+>
+> You can't switch from an annual billing cycle to a monthly cycle.
+
+When you change the billing cycle's duration:
+
+- The next billing date reflects the new cycle. To find your next billing date, see [View renewal date](history.md#view-renewal-date).
+- The subscription's start date resets. For example, if the start date of the monthly subscription is March 1st and the end date is April 1st, then after switching the billing duration to March 15th, 2023 the new start date is March 15th, 2023, and the new end date is March 15th, 2024.
+- Any unused monthly subscription is prorated and applied as credit towards the new annual period. For example, if you switch from a $10 monthly subscription to a $100 annual plan, deducting the unused monthly value (in this case $5), the migration cost becomes $95 ($100 - $5). The renewal cost after March 15, 2024 is $100.
```

Comment:
**nitpick**: if you do change the dates above, these dates should change as well.